### PR TITLE
CVE Records May 11th

### PR DESCRIPTION
**Pull request by monicamurray**

### CODE REVIEW

File: 2021/27xxx/CVE-2021-27772.json [hunk 1]
```diff
@@ -1,18 +1,92 @@
 {
```
Comment (cve-team) [inline 2021/27xxx/CVE-2021-27772.json:1]: 'value' is a required property

File: 2021/27xxx/CVE-2021-27773.json [hunk 2]
```diff
@@ -1,18 +1,92 @@
 {
```
Comment (cve-team) [inline 2021/27xxx/CVE-2021-27773.json:1]: 'value' is a required property

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2021/27xxx/CVE-2021-27772.json (hunk 1 above)
Comment (cve-team) [inline 2021/27xxx/CVE-2021-27772.json:1]: 'value' is a required property

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2021/27xxx/CVE-2021-27773.json (hunk 2 above)
Comment (monicamurray) [reply 2021/27xxx/CVE-2021-27773.json:1]: Forgot to delete the extra field which was not needed.

**Pull request merged**